public/js/common/export/export.js

```diff
@@ -37,8 +37,10 @@ export var ExportModal = React.createClass({
     },
 
     getGraphImage: function() {
+        var JsonLocalStorageObj = JSON.stringify(localStorage);
```

Comment:
This is a good point. I believe local storage values can themselves be dictionaries, so it would be better even to just put all the course info under a nested "courses" dictionary in the local storage. (You could create/modify a separate helper for accessing them.)